tensorflow_recommenders_addons/dynamic_embedding/python/ops/dynamic_embedding_variable.py

```diff
@@ -344,6 +344,56 @@ def upsert(self, keys, values, name=None):
 
     return control_flow_ops.group(ops_)
 
+  def accum(self, keys, old_values, new_values, exists, name=None):
+    """
+    Insert `keys` with `values` if not exist, or Accumlate a delta value
+      `new_values - old_values` to 'keys'.
+    This API will help relieve stale gradient problem in asynchronous training.
+
+    Args:
+      keys: Keys to insert. Can be a tensor of any shape. Must match
+        the table's key type.
+      old_values: old values to be associated with keys. Must be a tensor of the
+        same shape as `keys` and match the table's value type.
```

Comment:
Seems that `old_values` and `new_values` do not have same shape as `keys`. Plz also check `upsert` method

Reply:
Here means Value  will be treat as an array/ matrix with ValueType, and I optimize the description a little.